types/types.go

```diff
@@ -34,6 +34,8 @@ const (
 	AlertStateUnprocessed AlertState = "unprocessed"
 	AlertStateActive      AlertState = "active"
 	AlertStateSuppressed  AlertState = "suppressed"
+	AlertStateDeleted     AlertState = "deleted"
+	// TODO(khellemun): add firing state
```

Comment:
isn't firing == active ?

Reply:
There is a case where it's marked as active, but hasn't fired yet. 
fired == notification has been send.

However, I'm not sure if the distinction between these 2 states is valuable.